app/assets/stylesheets/dance_comments.scss

```diff
@@ -0,0 +1,22 @@
+#comment_box {
+  padding: 20px;
+  margin: 5px;
+  border: 3px solid black;
+  position: relative;
+}
+
+#comment_box p {
+  padding-left: 5px;
+}
+
+#comment_box footer ul {
```

Comment:
I had forgotten about nested selectors. I feel like I'm having to reteach myself everything as I move from html to css to javascript to AJAX and back to basic rails/ruby. If I'm out of one for too long I need to refresh on it to be able to start anything new.